.forge/skills/de-sloppify/SKILL.md

```
@@ -0,0 +1,277 @@
+
+**Step 2: Remove Unused Usings**
+```bash
+dotnet format analyzers --diagnostics IDE0005
+```
+Why second: Unused usings are noise. Removing them makes subsequent analysis cleaner and reduces false positives in code review.
+
+Verify: `dotnet build` should show no new errors. Run `dotnet test` if usings removal is extensive.
+Commit: `chore: remove unused using statements`
+
+**Step 3: Fix Analyzer Warnings**
+```
```

markdownlint-cli2 (0.22.1)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)